src/node/entry.ts

```diff
@@ -79,14 +79,20 @@ const main = async (args: Args): Promise<void> => {
   const serverAddress = await httpServer.listen()
   logger.info(`HTTP server listening on ${serverAddress}`)
 
-  if (auth === AuthType.Password && !process.env.PASSWORD) {
-    logger.info(`  - Password is ${originalPassword}`)
-    logger.info("    - To use your own password set the PASSWORD environment variable")
-    if (!args.auth) {
-      logger.info("    - To disable use `--auth none`")
+  if (!args.auth) {
+    args = {
+      ...args,
+      auth: AuthType.Password,
```

Comment:
I think we'll want to set this default above since otherwise `HttpServer` gets the wrong value.

Reply:
Fixed.